build_scripts/git_shortlog.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Fetch name email information for linux kernel contributors
+set -e
+
+TAG="$1"
```

Comment:
What happens if the tag argument is not provided on the command line? You might consider checking for a value here and then calling a `usage()` function if none was provided. This is a pattern I occasionally use:

```suggestion
usage() {
    echo "Usage: $0 tag"
    exit 1
}

TAG="$1"

[[ -z ${TAG} ]] && usage
```

Reply:
Change was applied